src/store/ui-config/index.js

```diff
@@ -21,7 +22,7 @@ const DEFAULT = new $Map({
     links: [
         {
             value: 'Documentation',
-            icon: 'library_books',
+            icon: LibraryBooks,
```

Comment:
This is coming dynamically from the API and will override these defaults. See the comment above.